src/dart/ast.rs

```diff
@@ -121,56 +140,90 @@ pub enum Item {
         mixins: Vec<Node<Qualified>>,
         interfaces: Vec<Node<Qualified>>,
     },
+    /// Enumerated types are a special kind of class used to represent
+    /// a fixed set of constant values.
+    ///
+    /// (`enum Type { integer, boolean, string }`).
     Enum {
         meta: Meta,
         name: Symbol,
         values: Vec<(Meta, Symbol)>,
     },
+    /// A type alias defines a name for a given type
+    /// (`typedef int example(Object a, Object b);`).
     TypeAlias {
         meta: Meta,
         name: Symbol,
         generics: Vec<Node<TypeParameter>>,
         ty: Node<Type>,
     },
+    /// Function definition (`example(x) { return x }`).
     Function {
         meta: Meta,
         external: bool,
         function: Node<Function>,
     },
+    /// Global variables (`var example = 123`).
     Vars(Meta, VarType, Vec<Node<VarDef>>),
 }
 
+/// Specifies which items of the imported library to be shown/hidden.
 #[derive(Clone, Debug)]
 pub struct ImportFilter {
+    /// Specifies the action to be applied to the items specified.
     pub hide: bool,
+    /// Specifies the items of the library.
     pub names: Vec<Symbol>,
 }
 
+/// Specifies a library to be used in the scope of the current library.
+///
+/// `import 'dart:specifier';`
```

Comment:
`dart:io` is probably a better example here since it's short and used in real code.